docs/book/src/topics/spot-vms.md

```diff
@@ -48,6 +48,18 @@ spec:
       maxPrice: 0.04 # Price in USD per hour (up to 5 decimal places)
 ```
 
+In addition, you are able to explicitly set the eviction policy for the Spot VM.
+The default policy is `Delete` which will delete the VM when it is evicted. You
```

Comment:
should that say "The default policy is `Deallocate`"? https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2689/files#diff-2128b31583d63daf5c4f86f34b7f90859ec634490812674658d179694a88bf2bR44